package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dosomething/analytics",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Let's do this with `npm version minor`: 

1. It's a minor release since it adds functionality (a new API).
2. We want a git tag so we can look at release history in GitHub!